mopidy_alsamixer/mixer.py

```diff
@@ -173,36 +226,44 @@ def trigger_events_for_changed_values(self):
             self.trigger_mute_changed(self._last_mute)
 
 
-class AlsaMixerObserver(threading.Thread):
-    daemon = True
-    name = "AlsaMixerObserver"
+class AlsaMixerObserver(PollingActor):
```

Comment:
You could set `use_daemon_thread = True` as a class variable here to make sure this thread will not block the application from exiting.

Ref https://pykka.readthedocs.io/en/stable/runtimes/threading/#pykka.ThreadingActor.use_daemon_thread